corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -28,6 +28,9 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
         <div aria-label="<%- description %>" tabindex="0" class="query-description"><%= description %></div>
       <% } %>
     </div>
+    <% if (actions || isMultiSelect) { %>
```

Comment:
Does this condition want to be `&&` or maybe something else more specific? This seems like it will always show a second copy of case list actions, including "Search Again" or "Form Accessible from Case List".

Reply:
This is an open question at the moment. Waiting for delivery feedback.

Reply:
We have confirmation that we are only dealing with `Continue` action for now.